src/DI/MigrationsExtension.php

```diff
@@ -114,13 +114,15 @@ public function beforeCompile(): void
 		// Register console helper only if console is provided
 		$application = $builder->getByType(Application::class, false);
 
-		if ($application !== null) {
-			/** @var ServiceDefinition $applicationDef */
-			$applicationDef = $builder->getDefinition($application);
-			$applicationDef->addSetup(
-				new Statement('$service->getHelperSet()->set(?)', [$this->prefix('@configurationHelper')])
-			);
+		if ($application === null) {
```

Comment:
Thanks @Roman3349. Could you please drop codesniffer rule for ternary operator? I prefer to let it on user.

Reply:
@f3l1x Should I drop codesniffer early exit rule in [ninjify/qa](https://github.com/ninjify/qa) or just here?

Reply:
Please early return rule.